climada/engine/__init__.py

```diff
@@ -20,3 +20,4 @@
 """
 from .impact import *
 from .cost_benefit import *
+from .supplychain import *
```

Comment:
What do you mean by submodule of engine? Isn't it what it currently is? Anyway I just left were it was..

Reply:
Make a folder called `supplychain` and add the file `supplychain.py` in there.

Reply:
Mh I see but supplychain is only one file.. shall we need a submodule for one func (for unceratinty, for instance, there were more files)? Anyway it is fine for me, I think it is more a decision on when we want to make new submodules in climada and when not.

Reply:
Let me add @emanuel-schmid to the conversation. I think it is cleaner to make submodules, even if they have only a single file. Because `supplychain` or `uncertainty`  are simply not on the same level as `impact` or `costbenefit`. @emanuel-schmid , what do you think?

Reply:
As long as there is only one file belonging to this module, `climada/engine/supplychain.py` should be fine I guess.